grpc-api-client/src/main/java/io/deephaven/grpc_api/DefensiveCapture.java

```diff
@@ -0,0 +1,59 @@
+package io.deephaven.grpc_api;
+
+import io.deephaven.grpc_api_client.util.BarrageProtoUtil.ExposedByteArrayOutputStream;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Objects;
+
+class DefensiveCapture extends InputStream {
```

Comment:
I believe this class and constructor should be public. It should be moved into package `grpc_api.util`.